napari/_vispy/utils/gl.py

```diff
@@ -161,4 +161,11 @@ def fix_data_dtype(data: npt.NDArray) -> npt.NDArray:
         'blend': True,
         'blend_equation': 'min',
     },
+    'multiplicative': {
+        'depth_test': False,
+        'cull_face': False,
+        'blend': True,
+        'blend_func': ('dst_color', 'zero', 'one', 'one'),
+        'blend_equation': 'func_add',
```

Comment:
@brisvag can you explain to me how this work? The `func_add` really threw me. 😂 

what are the contents of the `blend_func` tuple?

Maybe a comment here would be interesting.

Reply:
It's trippy, yeah. I learned from https://www.khronos.org/opengl/wiki/Blending#Blending_Parameters

Maybe I'll just put a link, cause this requires some background.

The relevant bits are:

Blending equation:
```
* Orgb = srgb * Srgb + drgb * Drgb
* Oa = sa * Sa + da * Da
```

(O = origin, D = destination, small is parameter, capitalized is the value itself)

"For example, if the blending you want is a straight multiplication of the source and destination colors, O = S * D, then you should set the GL_FUNC_ADD function. Then set the source parameter for that equation to GL_DST_COLOR, and set the destination parameter to GL_ZERO (we don't want to do addition, so null out the additive term). This will give O = D * S + 0 * D. "